docs/Reference/Collection/Indexes.md

```diff
@@ -75,8 +84,11 @@ Creates a Geo index on the collection.
 
 * **opts**: `Object` (optional)
 
-  Additional options for this index. If the value is a boolean, it will be
-  interpreted as `opts.unique`.
+  An object containing additional properties of the index.
+
+    * **geoJson**: `string` 
+
+      If a geo-spatial index on a location is constructed and geoJson is true, then the order within the array is longitude followed by latitude. This corresponds to the format described in http://geojson.org/geojson-spec.html#positions.
```

Comment:
check indentation how it looks in docs.

Reply:
Looks good 

<img width="1022" alt="Screenshot 2021-09-27 at 1 05 55 PM" src="https://user-images.githubusercontent.com/83563339/134864489-5695ef67-f2df-4d15-9510-c74c3d726d02.png">